docs/usage_and_configuration/black_docker_image.md

```diff
@@ -0,0 +1,40 @@
+# Black Docker image
+
+Official _Black_ Docker images are available on Docker Hub:
+https://hub.docker.com/r/pyfound/black
+
+Following tags are used for _Black_ images:
+
+- `latest` - tag used for either
```

Comment:
I don't understand what this would mean to a user. When is it one versus the other?

Reply:
It's quite hard to express the difference without going too much into technical details.

I tried to address it via bd03bf8. Human-readable version to be found here:  
https://github.com/pszlazak/black/blob/bd03bf86ff76f930b0f2bc14f7ee01891e267acf/docs/usage_and_configuration/black_docker_image.md

Reply:
Thanks! Can't we just say it's from the latest commit to main? Releases of black are always made from main.

Reply:
You mean `latest`?
- `latest` is created when there is new commit or new tag (and tags are created during release):  
https://github.com/psf/black/blob/b1d060101626aa1c332f52e4bdf0ae5e4cc07990/.github/workflows/docker.yml#L41
- `latest_non_release` is created when commit does not map to tag:
https://github.com/psf/black/blob/b1d060101626aa1c332f52e4bdf0ae5e4cc07990/.github/workflows/docker.yml#L32
https://github.com/psf/black/blob/b1d060101626aa1c332f52e4bdf0ae5e4cc07990/.github/workflows/docker.yml#L33
- `latest_release` is created during realease:
    - condition:
https://github.com/psf/black/blob/b1d060101626aa1c332f52e4bdf0ae5e4cc07990/.github/workflows/docker.yml#L44
    - creation:
https://github.com/psf/black/blob/b1d060101626aa1c332f52e4bdf0ae5e4cc07990/.github/workflows/docker.yml#L50

Reply:
I guess from end user perspective only two tags are important:
- `latest` - always the newest version of _Black_ (bleeding edge)
- `latest_release` - latest "stable" version

Third option - if anyone wants to use specific version of _Black_, image tags corresponding to Git tags can be used.

Reply:
@JelleZijlstra - please take a look at e25cafe. Human-readable version to be found here:  
https://github.com/pszlazak/black/blob/e25cafe/docs/usage_and_configuration/black_docker_image.md

I hope I finally found sweet-spot between being to technical and end-user friendly.